cars/v1/lean-watermarks.ini

```diff
@@ -0,0 +1,11 @@
+[meta]
+description=Configure lean watermark levels (useful in testing with little disk resources)
+type=mixin
+
+[config]
+base=lean_watermarks
+
+[variables]
+#watermark_low=500mb
+#watermark_high=300mb
+#watermark_flood_stage=100mb
```

Comment:
Why are those variables seemingly commented out?

Reply:
These are the defaults.